frontend/src/pages/Friends.jsx

```diff
@@ -31,26 +31,37 @@ class Friends extends Component {
 	}
 	
 	componentDidMount(){
-		let userIdShortString = utils.getShortAuthorId(Cookies.get("userID")) || store.getState().loginReducers.authorId;
-		let userIdString = Cookies.get("userID") || store.getState().loginReducers.userId;
-		let hostNameString = utils.getHostName(Cookies.get("userID")) || store.getState().loginReducers.hostName;
+		try{
+			let userIdShortString = utils.getShortAuthorId(Cookies.get("userID")) || store.getState().loginReducers.authorId;
+			let userIdString = Cookies.get("userID") || store.getState().loginReducers.userId;
+			let hostNameString = utils.getHostName(Cookies.get("userID")) || store.getState().loginReducers.hostName;
 
-		if(userIdShortString === null || userIdString === null || hostNameString === null){
-			console.error("Error: Login credentials expired")
-			return null
-		}
+			if(userIdShortString === null || userIdString === null || hostNameString === null){
```

Comment:
Instead of checking if they are null, take advantage of javascript's type values and compare useridShortString || instead since nulls is a falsey value. If there is something in the variable, it will return true.